pkg/api/converterfromapi.go

```diff
@@ -285,6 +285,17 @@ func convertOrchestratorProfileToV20160330(api *OrchestratorProfile, o *v2016033
 
 func convertOrchestratorProfileToVLabs(api *OrchestratorProfile, o *vlabs.OrchestratorProfile) {
 	o.OrchestratorType = vlabs.OrchestratorType(api.OrchestratorType)
+
+	if api.KubernetesConfig != nil {
+		o.KubernetesConfig = &vlabs.KubernetesConfig{}
+		convertKubernetesConfigToVLabs(api.KubernetesConfig, o.KubernetesConfig)
+	}
+}
+
+func convertKubernetesConfigToVLabs(api *KubernetesConfig, vlabs *vlabs.KubernetesConfig) {
+	vlabs.DnsServiceIp = api.DnsServiceIP
```

Comment:
seems like this is missing the inverse to these lines from convertertoapi.go

api.KubernetesImageBase = vlabs.KubernetesImageBase		  	
api.NetworkPolicy = vlabs.NetworkPolicy

could you add them please